prac_06/guitar.py

```diff
@@ -0,0 +1,30 @@
+"""
+File: language.py
+Start Time: 9:10 PM
+Finish Time: 10:45 PM
+"""
+
+CURRENT_YEAR = 2024
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """class for storing  of a guitar."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return a string representation of a Guitar."""
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
```

Comment:
The __str__ method provides a clear and formatted string representation of the Guitar object, which is useful for displaying information.